lib/chai/utils/inspect.js

```diff
@@ -318,6 +352,12 @@ function reduceToSingleString(output, base, braces) {
   return braces[0] + base + ' ' + output.join(', ') + ' ' + braces[1];
 }
 
+function isTypedArray(ar) {
+  // Unfortunately there's no way to check if an object is a TypedArray
+  // We have to check if it's one of these types
```

Comment:
Yep! 😄 
The goal of the notes I'm adding here is for me to remember what to do later, they aren't necessarily related to the `4.x.x` merge.
I'm just posting them to keep track of what should be done hehe